docs/source/release/v6.0.0/muon.rst

```diff
@@ -5,10 +5,6 @@ MuSR Changes
 .. contents:: Table of Contents
    :local:
 
-.. warning:: **Developers:** Sort changes under appropriate heading
-    putting new features at the top of the section, followed by
-    improvements, followed by bug fixes.
-
 Muon Analysis 2 and Frequency Domain Analysis
 ---------------------------------------------
 
```

Comment:
I think it's worth removing the `New Features` title on line 11, OR merging it worth improvements to make `New & Improved`?